cobigen/cobigen-core-systemtest/src/test/java/com/devonfw/cobigen/systemtest/TemplateProcessingTest.java

```diff
@@ -100,6 +101,7 @@ public void extractTemplateSetsTest() throws IOException {
    * @throws IOException if an Exception occurs
    */
   @Test
+  @Ignore
```

Comment:
Why did you disable this test?

Reply:
This test does fit the second critiria in the ticket. I will mark all tests at the beginning with Ignore, then delete them. Although I am not exactly sure if the test should be deleted.

Reply:
@maybeec @jan-vcapgemini Do you think this test should be deleted, Because it is redundant to what is implemented in core?